<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_staff_form</class>
 <widget class="QWidget" name="view_staff_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>335</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>view_staff</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="display_mgt_package_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>424</width>
        <height>363</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QScrollArea" name="staff_member_list">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>198</width>
            <height>343</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">


background-color: rgb(199, 156, 153);
</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>302</width>
            <height>537</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QFrame" name="frame_18">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>124</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="staff_member_pp">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>80</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(168, 168, 168);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>18</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_21">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Staff id</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="view_s_id">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>S001</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_22">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="view_s_name">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>Staff Member Name</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_23">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Gander</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_20">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Department</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_24">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Address</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QTextEdit" name="view_s_address">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>80</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QTextEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>Staff Member's Address</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_25">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Tel. Number</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="view_s_tel">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>07X XX XX XXX</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_26">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>E-mail</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="view_s_email">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>emample : xyz@gmail.com</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_27">
               <property name="styleSheet">
                <string notr="true">font: 700 10pt &quot;SF Pro Display&quot;;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>Salary</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="view_s_salary">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>Rs. 45000.00</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="view_s_gender">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>example : Male</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="view_s_department">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #4C4C6C;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>example : Maintenance</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QFrame" name="frame_21">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>5</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="view_s_update">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">/* Default button style */
QPushButton {
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #00000;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 

}

/* Button style on hover */
QPushButton:hover {
 background-color: #46464b;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #AE544F;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #AE544F;
 	color: #FFFFFF;
}</string>
                </property>
                <property name="text">
                 <string>Update</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_10">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>31</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="view_s_delete">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">/* Default button style */
QPushButton {
    background-color: #4C4C6C;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #ffffff;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 



}

/* Button style on hover */
QPushButton:hover {
 background-color: #46464b;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #AE544F;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #AE544F;
 	color: #FFFFFF;
}</string>
                </property>
                <property name="text">
                 <string>Delete</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_11">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>32</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
